### PR TITLE
Wip add modulename √

### PR DESCRIPTION
**Pull request by viktorklang**
Including @kiiadi's commit here.

### CODE REVIEW

File: flow-adapters/build.gradle [hunk 1]
```diff
@@ -6,8 +6,21 @@ dependencies {
     testCompile project(':reactive-streams-tck')
     testCompile group: 'org.testng', name: 'testng', version: '5.14.10'
 }
+
+jar {
```
Comment (akarnokd) [inline flow-adapters/build.gradle]: This appears twice.

**Comment by viktorklang** [comment]
@kiiadi @reactive-streams/contributors Everyone OK with this?

File: flow-adapters/build.gradle (hunk 1 above)
Comment (viktorklang) [reply flow-adapters/build.gradle]: @akarnokd Nice catch, I've fixed that now :)

File: flow-adapters/build.gradle [hunk 2]
```diff
@@ -6,8 +6,21 @@ dependencies {
...
+    manifest {
+        attributes('Automatic-Module-Name': 'org.reactivestreams.flowadapters')
+    }
+}
+
 test.useTestNG()
 
+jar {
```
Comment (akarnokd) [inline flow-adapters/build.gradle]: Again, but with different name?

File: flow-adapters/build.gradle [hunk 3]
```diff
@@ -6,8 +6,21 @@ dependencies {
...
+    manifest {
+        attributes('Automatic-Module-Name': 'org.reactivestreams.flowbridge')
+    }
+}
+
 javadoc {
     options.links("http://download.java.net/java/jdk9/docs/api")
```
Comment (akarnokd) [inline flow-adapters/build.gradle]: This gives 404. The proper link is `https://docs.oracle.com/javase/9/docs/api/`

**Comment by ktoso** [comment]
Checked the names, this looks good to go 👍

**Comment by viktorklang** [comment]
Merging!

**Pull request merged**